### ISSUE TITLE
Stuck at "Finding online channel..." too

### ISSUE THREAD

**Issue by wodepig** [AlexSterk/TwitchDropGrabber#30]
Thank you for making a very useful tool.I encountered some small problems during use.
I have seen this [issue](https://github.com/AlexSterk/TwitchDropGrabber/issues/23), but our problem is different.
I modified the url and added the log.
![image](https://user-images.githubusercontent.com/42103087/144341342-751a596a-c35c-4cea-bc98-1f3311443df3.png)
There is an online channel after using the browser to access the url
![image](https://user-images.githubusercontent.com/42103087/144341523-349074f3-4eec-422b-9a92-9e52431c344b.png)
I used the -v command but it showed exactly the same output.
![image](https://user-images.githubusercontent.com/42103087/144341677-24005d0b-7e50-44cc-95f1-d926c08a9459.png)

Will you help me with this problem? thanks

**Comment by AlexSterk** [comment]
So if I'm understanding correctly, the program will open the list of channels, but not select a channel?

I tried to apply the changes you made (removing the Drops Enabled tag from the directoryUrl), but I did not have the problem you are describing. It just works for me.

Can you try with the `--no-headless` option and see what Chrome does?

**Comment by wodepig** [comment]
thank you for your reply
After I used the command you provided, I found that there was no data after entering the channel. I checked and found that there was a link that reported an error.
![before](https://user-images.githubusercontent.com/42103087/144352003-e8e219a8-4dbd-4a61-9042-717c4225324d.png)
He prompts xxxuser, then I log in to my twitch normally, and after re-obtaining the auth-token, I found that the hair has changed (I didn't change the password)
![after](https://user-images.githubusercontent.com/42103087/144351863-120cb1fa-e453-40ec-bdec-b95a6b791e5e.png)
Everything is normal after updating the variables.. It may be that my network problem has caused the auth-token to change
In short, thank you very much for your reply, it would be better if the software can detect the 401 status and give a prompt, thank you

**Comment by wodepig** [comment]
国内的朋友一直卡在获取频道的话可以检查下你的auth-token是不是变了,可能你的vpn换了个节点就会导致这个问题(我的就是这样)
